src/test/resources/uk/gov/ons/ctp/integration/rhsvc/endpoint/UniqueAccessCodeEndpointTest.UniqueAccessCodeDTO.json

```diff
@@ -2,18 +2,39 @@
 	"uacHash": "8a9d5db4bbee34fd16e40aa2aaae52cfbdf1842559023614c30edb480ec252b4",
 	"active": true,
 	"qid": 1110000009,
-	"region": "E",
-	"caseId": "dc4477d1-dd3f-4c69-b181-7ff725dc9fa4",
-	"collectionExerciseId": "a66de4dc-3c3b-11e9-b210-d663bd873d93",
-	"address": {
-		"addressLine1": "1 main street",
-		"addressLine2": "upper upperingham",
-		"addressLine3": "",
-		"townName": "upton",
-		"postcode": "UP103UP",
-		"uprn": "123456"
-	},
 	"receiptReceived": "true",
 	"eqLaunched": "true",
-	"wave": "82"
+	"wave": "82",
+	"survey": {
+		"surveyId": "3883af91-0052-4497-9805-3238544fcf8a",
+		"name": "LMS"
+	},
+	"collectionExercise": {
+		"collectionExerciseId": "4883af91-0052-4497-9805-3238544fcf8a",
+		"surveyId": "3883af91-0052-4497-9805-3238544fcf8a",
+		"name": "velit",
+		"reference": "MVP012021",
+		"startDate": "2021-09-17T23:59:59.999Z",
+		"endDate": "2021-09-27T23:59:59.999Z",
+		"numberOfWaves": "3",
+		"waveLength": "2",
+		"cohorts": "3",
+		"cohortSchedule": "7"
+	},
+	"collectionCase": {
+		"caseId": "dc4477d1-dd3f-4c69-b181-7ff725dc9fa4",
+		"surveyId": "3883af91-0052-4497-9805-3238544fcf8a",
+		"collectionExerciseId": "4883af91-0052-4497-9805-3238544fcf8a",
+		"invalid": false,
+		"refusalReceived": "HARD_REFUSAL",
+		"address": {
+			"addressLine1": "1 main street",
+			"addressLine2": "upper upperingham",
+			"addressLine3": "",
+			"townName": "upton",
+			"region": "E",
+			"postcode": "UP103UP",
+			"uprn": "123456"
+		}
+	}
 }
```

Comment:
newline please. Also please format consistently.  We generally have 2-space indent for JSON files.